code/modules/research/designs/guncrafting_designs.dm

```diff
@@ -0,0 +1,544 @@
+	name = "Gun Energy Modifier: Disorienter"
+	desc = "A disorienter mod."
+	id = "ebullet_disorient"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300)
+	build_path = /obj/item/weapon/gun_attachment/energy_bullet/disorienter
+	category = list("Guncrafting Parts")
+
+/datum/design/ebullet_decloner
+	name = "Gun Energy Modifier: Decloner"
+	desc = "A decloner mod."
+	id = "ebullet_decloner"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300, MAT_URANIUM = 2000)
+	build_path = /obj/item/weapon/gun_attachment/energy_bullet/decloner
+	category = list("Guncrafting Parts")
+
+/datum/design/ebullet_speed
+	name = "Gun Energy Modifier: Speed"
+	desc = "A speed mod."
+	id = "ebullet_speed"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300)
+	build_path = /obj/item/weapon/gun_attachment/energy_bullet/speed
+	category = list("Guncrafting Parts")
+
+/datum/design/bullet_rad
+	name = "Gun Bullet Modifier: Polonium"
+	desc = "A polonium mod."
+	id = "bullet_rad"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300, MAT_URANIUM = 2000)
+	build_path = /obj/item/weapon/gun_attachment/bullet/polonium
+	category = list("Guncrafting Parts")
+
+/datum/design/bullet_fire
+	name = "Gun Bullet Modifier: Incendiary"
+	desc = "A fire mod."
+	id = "bullet_fire"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300)
+	build_path = /obj/item/weapon/gun_attachment/bullet/fire
+	category = list("Guncrafting Parts")
+
+/datum/design/bullet_bleed
+	name = "Gun Bullet Modifier: Haemorrhage-Inflicting"
+	desc = "A bleeding mod."
+	id = "bullet_bleed"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300)
+	build_path = /obj/item/weapon/gun_attachment/bullet/haemorrhage
+	category = list("Guncrafting Parts")
+
+/datum/design/bullet_pen
+	name = "Gun Bullet Modifier: Penetrator"
+	desc = "A penetrator mod."
+	id = "bullet_pen"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300)
+	build_path = /obj/item/weapon/gun_attachment/bullet/penetrator
+	category = list("Guncrafting Parts")
+
+//scopes
+/datum/design/scope_reflex
+	name = "Gun Sight: Reflex"
+	desc = "A reflex sight."
+	id = "scope_reflex"
+	req_tech = list("combat" = 1, "materials" = 1)
+	build_type = GUNLATHE
+	materials = list(MAT_METAL = 500, MAT_GLASS = 300)
+	build_path = /obj/item/weapon/gun_attachment/scope/reflex
+	category = list("Guncrafting Parts")
+
+/datum/design/scope_rd
+	name = "Gun Sight: Red-Dot"
+	desc = "A red-dot sight."
+	id = "scope_rd"
+	req_tech = list("combat" = 1, "materials" = 1)
```

Comment:
This scope should at the very least be combat 2 and mat 2.Since it seems silly being able to get all the scopes roundstart